README.md

```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
```

Comment:
Merge all the changes that need to go into the release to the master branch.

Reply:
Done.